modules/influxdb/src/test/java/org/testcontainers/containers/InfluxDBContainerTest.java

```diff
@@ -1,44 +1,137 @@
 package org.testcontainers.containers;
 
-import org.influxdb.InfluxDB;
-import org.junit.ClassRule;
+import com.influxdb.client.InfluxDBClient;
+import com.influxdb.client.QueryApi;
+import com.influxdb.client.WriteApi;
+import com.influxdb.client.domain.Bucket;
+import com.influxdb.client.domain.WritePrecision;
+import com.influxdb.client.write.Point;
+import com.influxdb.query.FluxRecord;
+import com.influxdb.query.FluxTable;
+import org.jetbrains.annotations.Nullable;
+import org.junit.After;
 import org.junit.Test;
 
+import java.time.Instant;
+import java.util.List;
+
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class InfluxDBContainerTest {
 
-    @ClassRule
-    public static InfluxDBContainer<?> influxDBContainer = new InfluxDBContainer<>(
-        InfluxDBTestImages.INFLUXDB_TEST_IMAGE
-    );
+    private static final String USERNAME = "new-test-user";
+
+    private static final String PASSWORD = "new-test-password";
+
+    private static final String ORG = "new-test-org";
+
+    private static final String BUCKET = "new-test-bucket";
+
+    private static final String RETENTION = "1w";
+
+    private static final String ADMIN_TOKEN = "super-secret-token";
+
+    @Nullable
+    private InfluxDBClient influxDBClient = null;
+
+    @After
+    public void stopInfluxDBClient() {
+        if (this.influxDBClient != null) {
+            this.influxDBClient.close();
+            this.influxDBClient = null;
+        }
+    }
 
     @Test
     public void getUrl() {
-        String actual = influxDBContainer.getUrl();
+        try (
+            final InfluxDBContainer influxDBContainer = new InfluxDBContainer(InfluxDBTestUtils.INFLUXDB_V2_TEST_IMAGE)
+        ) {
+            influxDBContainer.start();
+            assertThat(influxDBContainer.isRunning()).isTrue();
```

Comment:
The `isRunning()` asserts should be superfluous, the container should always be running after successfully proceeding after a `start()` call.